src/js/Properties.js

```diff
@@ -703,8 +703,7 @@ axs.properties.getAriaProperties = function(element) {
  */
 axs.properties.getGlobalAriaProperties = function(element) {
     var globalProperties = {};
-    for (var i = 0; i < axs.constants.GLOBAL_PROPERTIES.length; i++) {
-        var property = axs.constants.GLOBAL_PROPERTIES[i];
+    for (var property in axs.constants.GLOBAL_PROPERTIES) {
```

Comment:
i'd consider adding ` && axs.constants.GLOBAL_PROPERTIES.hasOwnProperty(property)` but it's not truly necessary here